editoast/src/views/speed_limit_tags.rs

```diff
@@ -0,0 +1,47 @@
+use actix_web::get;
+use actix_web::web::Data;
+use actix_web::web::Json;
+
+use crate::error::Result;
+use crate::generated_data::speed_limit_tags_config::SpeedLimitTagIds;
+
+crate::routes! {
+    "/speed_limit_tags" => {
+        speed_limit_tags,
+    },
+}
+
+#[utoipa::path(
+    tag = "speed_limit_tags",
+    responses(
+        (status = 200, description = "List of configured speed-limit tags", body = Vec<String>, example = json!(["V200", "MA80"])),
+    ),
+)]
+#[get("")]
+async fn speed_limit_tags(
+    speed_limit_tag_ids: Data<SpeedLimitTagIds>,
+) -> Result<Json<SpeedLimitTagIds>> {
+    Ok(Json((**speed_limit_tag_ids).clone()))
```

Comment:
Maybe this is less weird to understand?

```suggestion
    Ok(Json(speed_limit_tag_ids.get_ref().clone()))
```

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/8106/commits/bf8e3fecc408e6d398b2e581f4f49afa2e222d69